scripts/coverage_diff.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Install Intel FPGA SDK for OpenCL Pro Edition
+# https://fpgasoftware.intel.com/
+
+# Example usage: python scripts/coverage_diff.py child_build_dir parent_build_dir
```

Comment:
This would execute the script with a Python2 interpreter on many systems. How about
```python
 # Example usage: ./coverage_diff.py child_build_dir parent_build_dir
```
No need to duplicate the directory, script might be moved to another directory later then this is outdated.